student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person' # Require the person.rb file
+
+class Student < Person
+  def initialize(classroom, name: 'Unknown', age: null, parent_permission: true)
+    super(name: name, age: age, parent_permission: parent_permission)
```

Comment:
- [ ] Based on earlier feedback, the `age` should not be optional. It should always be provided:

  ```
  def initialize(classroom, age, name: 'Unknown', parent_permission: true)
    super(age, name: name, parent_permission: parent_permission)
    ...
  ```